README.md

```diff
@@ -1 +1,138 @@
+case for this image is to start Accumulo processes (i.e tserver, master, etc), you can run other
+commands in the `accumulo` script to test out the image:
+
+```bash
+# No arguments prints Accumulo command usage
+docker run accumulo
+# Print Accumulo version
+docker run accumulo version
+# Print Accumulo classpath
+docker run accumulo classpath
+```
+
+# Run Accumulo using Docker
+
+Before you can run Accumulo services in Docker, you will need to install Accumulo, configure `accumulo-site.xml`,
+and initialize your instance with `--upload-accumulo-site`. This will upload configuration to Zookeeper and limit
+how much configuration needs to be set on the command line.
+
+```bash
+$ accumulo init --upload-accumulo-site
+...
+Uploading properties in accumulo-site.xml to Zookeeper. Properties that cannot be set in Zookeeper will be skipped:
+Skipped - instance.secret = <hidden>
+Skipped - instance.volumes = hdfs://localhost:8020/accumulo
+Skipped - instance.zookeeper.host = localhost:2181
+Uploaded - table.durability = flush
+Uploaded - tserver.memory.maps.native.enabled = false
+Uploaded - tserver.readahead.concurrent.max = 64
+Uploaded - tserver.server.threads.minimum = 64
+Uploaded - tserver.walog.max.size = 512M
+```
+
+Any configuration that is skipped above will need to be passed in as a command line option to Accumulo services running
+in Docker containers. These options can be set in an environment variable which is used in later commands.
+
+```
+export ACCUMULO_CL_OPTS="-o instance.secret=mysecret -o instance.volumes=hdfs://localhost:8020/accumulo -o instance.zookeeper.host=localhost:2181"
+```
+
+The Accumulo docker image expects that the HDFS path set by `instance.volumes` is owned by the `accumulo` user. This
+can be accomplished by running the command below (replace the HDFS path with yours):
+
+```bash
+hdfs dfs -chown -R accumulo hdfs://localhost:8020/accumulo
+```
+
+## Docker engine
+
+Use the `docker` command to start local docker containers. The commands below will start a local Accumulo cluster
+with two tablet servers.
+
+```
+docker run -d --network="host" accumulo monitor $ACCUMULO_CL_OPTS
+docker run -d --network="host" accumulo tserver $ACCUMULO_CL_OPTS
+docker run -d --network="host" accumulo tserver $ACCUMULO_CL_OPTS
+docker run -d --network="host" accumulo master $ACCUMULO_CL_OPTS
+docker run -d --network="host" accumulo gc $ACCUMULO_CL_OPTS
+```
+
+## Marathon
+
+Using the Marathon UI, you can start Accumulo services using the following
+JSON configuration template.  The template is configured to start an Accumulo
+monitor but it can be modified to start other Accumulo services such as
+`master`, `tserver` and `gc`. For tablet servers, set `instances` to the number
+of tablet servers that you want to run.
+
+```
+{
+  "id": "accumulo-monitor",
+  "cmd": "accumulo monitor -o instance.secret=mysecret -o instance.volumes=hdfs://localhost:8020/accumulo -o instance.zookeeper.host=localhost:2181",
+  "cpus": 1,
+  "mem": 512,
```

Comment:
Need a good way to set JVM mem.

Reply:
I think this can be done by settting ACCUMULO_JAVA_OPTS before you run docker command.

https://stackoverflow.com/questions/29923531/how-to-set-java-heap-size-xms-xmx-inside-docker-container

Reply:
That may be worth mentioning in the readme, how the JVM memory settings can be set and why it matters.